td-agent/Rakefile

```diff
@@ -324,7 +324,7 @@ class BuildTask
         configs = [
           "etc/td-agent/td-agent.conf",
           "etc/td-agent/td-agent.conf.tmpl",
-          "etc/logrotate.d/td-agent.logrotate",
+          "etc/logrotate.d/td-agent",
```

Comment:
The original td-agent package copies /etc/logrotate.d/td-agent.logrotate to /etc/logrotate.d/td-agent if it's not exist: https://github.com/treasure-data/omnibus-td-agent/blob/master/templates/package-scripts/td-agent/rpm/post#L45

So the patch is no problem if it doesn't replace the old one, but it seems that `noreplace` isn't specified in the spec file: https://github.com/clear-code/td-agent-builder/blob/d99d91f6c87fb9f6a3ba2de3fb0f9f45a1ad4ad9/td-agent/yum/td-agent.spec.in#L115

Reply:
@ashie Thanks,
I've changed to add `noreplace` for it.